Resources/doc/index.md

```diff
@@ -136,3 +136,14 @@ services:
       tags:
         -  { name: "console.command" }
 ```
+
+## User impersonation
+
+Add a `switch_user` to the firewall to enable user impersonation
+
+```yaml
+security:
+    firewalls:
+        main:
+          switch_user: { role: ROLE_ALLOWED_TO_SWITCH, parameter: _switch_user }
```

Comment:
is the default in symfony, could be changed to switch_user
http://symfony.com/doc/current/cookbook/security/impersonating_user.html